database/index.js

```diff
@@ -79,9 +79,76 @@ const deleteFromDatabase = (id) => {
   ProductInfo.remove({id}, function(err) { // TO RESET COLLECTION
     console.log('collection removed')
   });
-}
+};
 
 module.exports.updateDatabase = updateDatabase;
```

Comment:
Code convention is to put module.exports at the bottom of your file. 
I recommend updating this to ES6 syntax: `module.exports = {updateDatabase, queryDatabase, queryAllFromDatabase, deleteFromDatabase};` or `export { func1, func2, func3 };`.